styles.css

```diff
@@ -0,0 +1,366 @@
+
+
+h1 {
+  font-family: 'Bitter', serif;
+  color: #093C60;
+}
+
+h2 {
+  font-family: 'Indie Flower', cursive;
+  color: #81B742;
+}
+
+h3 {
+  font-family: 'Bitter', sans-serif;
+  color: #093C60;
+}
+
+h3.footer-h3 {
+  color: #ECECEC;
+  font-family: 'Muli', sans-serif;
+  line-height: 1.5;
+}
+
+h4 {
+  font-family: 'Bitter', sans-serif;
+  color: #093C60;
+  line-height: 0.5;
+}
+
+p {
+  color: #373943;
+  font-family: 'Muli', sans-serif;
+  line-height: 1.5;
+}
+
+p.small {
+  color: #373943;
+  font-family: 'Muli', sans-serif;
+  line-height: 1.5;
+  font-size: 9pt;
```

Comment:
If you're using a modifier class after an initial declaration of that element you can omit the repetiivce code, and just put what changes. In this case if you could get by by only using the font size.

Don't use pt for font sizes.  Points are much like pixels, in that they are fixed-size units and cannot scale in size.